charts/prometheus-node-exporter/values.yaml

```diff
@@ -2,7 +2,7 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 image:
-  repository: quay.io/prometheus/node-exporter
+  repository: quay.io/astronomer/node-exporter
```

Comment:
Should revert. There is no repository called astronomer/node-exporter, we are pulling from prometheus repo.